docs/src/PolyhedralGeometry/Polyhedra/visualization.md

```diff
@@ -0,0 +1,18 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg_polyhedra_visualize.md"]
```

Comment:
```suggestion
Pages = ["visualization.md"]
```
most of these were corrected in https://github.com/oscar-system/Oscar.jl/pull/708 but this is a new file.